connectors/src/connectors/github/index.ts

```diff
@@ -59,7 +59,7 @@ export async function createGithubConnector(
 
       return connector;
     });
-    await launchGithubFullSyncWorkflow(connector.id.toString());
+    await launchGithubFullSyncWorkflow(connector.id.toString(), false);
```

Comment:
can we make it an object params ? the dangling booleans are hard to read IMO

Reply:
def